### DISCUSSION TITLE
Change background color of Linux Desktop Client Dark Mode

### DISCUSSION THREAD

**Discussion by adrummer354** [pia-foss/desktop#33] [Ideas]
Would there be a way to change the color of the main background of the desktop client once it is changed to dark mode so that it matches my Linux Mint Y Dark theme? The theme color I am looking to change to is #2E2E2E and the PIA linux client currently runs in dark mode at #323642

https://imgur.com/iucTCWq

**Comment by JonathonH-PIA** [comment]
There are no settings for this in the client, but if you want to try building from source, have a look at `DashboardStyle.backgroundColor`: https://github.com/pia-foss/desktop/blob/master/client/res/components/theme/DashboardStyle.qml

Color schemes for the other windows are in other files in that directory.  Building from source is pretty easy on Linux, just install the dependencies per the readme and then `rake installer` (debug) or `rake installer VARIANT=release`
